screenpy/resolutions/matches.py

```diff
@@ -17,14 +24,21 @@ class Matches:
         )
     """
 
+    @property
+    def item_to_log(self) -> str:
```

Comment:
I think i'd prefer this to be called `pattern_to_log` :P

Reply:
Funny, I almost used that.  I think I was trying to find a universal name for the property that would work for all of the classes regardless of what the "item" is.  Not sure why I thought that was important.
Updating it.